src/app/services/visualization3D.js

```diff
@@ -114,15 +114,8 @@ const Visualization3D = () => {
       // bbox
       const bboxFolder = gui.addFolder('Bounding Box');
       bboxFolder.add(stackHelper.bbox, 'visible');
-      bboxFolder.addColor(stackHelper.bbox, 'color');
```

Comment:
did we want to turn the light off in this one as well?

Reply:
good question, this file is responsible for rendering the non-segmented images though so there is no cad model or particle light